runelite-client/src/main/java/net/runelite/client/plugins/worldmap/DungeonLocation.java

```diff
@@ -31,8 +31,13 @@
 @Getter
 enum DungeonLocation
 {
+
 	ABANDONED_MINE("Abandoned Mine", new WorldPoint(3439, 3232, 0)),
 	ABANDONED_MINE_SECRET("Abandoned Mine (secret entrance)", new WorldPoint(3452, 3244, 0)),
+	ALDARIN_WINE_CELLAR("Wine Cellar", new WorldPoint(1447, 2938, 0)),
+	ALDARIN_BACKSTAGE("Backstage", new WorldPoint(1475, 2927, 0)),
```

Comment:
The wiki calls it `Backstage entrance` which might be a tiny bit more descriptive? but idk

Reply:
You're right, I was kind of just going off the name of the issue. Fixed it.